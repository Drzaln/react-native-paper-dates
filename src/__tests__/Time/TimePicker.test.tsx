import renderer from 'react-test-renderer'
import TimePicker from '../../Time/TimePicker'
import en from '../../translations/en'
import { registerTranslation } from '../../translations/utils'

registerTranslation('en', en)

it('renders TimePicker', () => {
  const tree = renderer
    .create(
      <TimePicker
        locale="en"
        inputType="keyboard"
        focused="hours"
        hours={6}
        minutes={30}
        onChange={() => null}
        onFocusInput={() => null}
      />
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
