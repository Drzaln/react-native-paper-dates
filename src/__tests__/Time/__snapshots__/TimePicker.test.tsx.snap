// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimePicker 1`] = `null`;
